torch/_inductor/triton_heuristics.py

```diff
@@ -339,6 +339,7 @@ def _precompile_config(self, cfg: Config, warm_cache_only_with_cc: Optional[int]
                 "num_warps": compile_meta["num_warps"],
                 "num_stages": compile_meta["num_stages"],
                 "debug": compile_meta["debug"],
+                "enable_fp_fusion": config.triton.enable_fp_fusion,
```

Comment:
@Chillee pointed out this me. I wonder how this will play with user defined triton kernels? Should we exclude those from this setting?